xds/experimental/xds_experimental.go

```diff
@@ -24,9 +24,16 @@ package experimental
 
 import (
 	"google.golang.org/grpc/balancer"
+	"google.golang.org/grpc/resolver"
 	xdsbalancer "google.golang.org/grpc/xds/internal/balancer"
+	xdsresolver "google.golang.org/grpc/xds/internal/resolver"
 )
 
 func init() {
 	balancer.Register(xdsbalancer.NewBalancerBuilder())
+
+	// The resolver builder needs to be registered after the balancer builder is
```

Comment:
The register order shouldn't matter if we move `parseOnce` to resolver's `Build()`.

Reply:
Done.